lib/rules/custom-property-pattern/__tests__/index.js

```diff
@@ -125,3 +125,18 @@ testRule({
 		},
 	],
 });
+
+testRule({
+	ruleName,
+	customSyntax: 'postcss-scss',
+	config: ['$foo-color'],
+
+	accept: [
+		{
+			code: 'a { color: var($foo-color); }',
+		},
+		{
+			code: 'a { color: var(tokens.$foo-color); }',
```

Comment:
```suggestion
			code: 'a { color: var(tokens.$bar-color); }',
```

Let's use `bar` here to show that the rule ignores this construct outright, and it's not connected to the config.